mllib/src/main/scala/org/apache/spark/ml/r/PrefixSpanWrapper.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.ml.r
+
+import org.apache.spark.ml.fpm.PrefixSpan
+import org.apache.spark.sql.{DataFrame, Dataset}
+
+private[r] class PrefixSpanWrapper private (val prefixSpan: PrefixSpan) {
```

Comment:
Is this wrapper needed? Can't we directly call `findFrequentSequentialPatterns`on `PrefixSpan` object?

Reply:
You are right. I will remove 
```
private[r] class PrefixSpanWrapper private (val prefixSpan: PrefixSpan) {
  def findFrequentSequentialPatterns(dataset: Dataset[_]): DataFrame = {
    prefixSpan.findFrequentSequentialPatterns(dataset)
  }
}
```